### PR TITLE
[FIX] fix mrope_position for qwen2.5VL

### PR DESCRIPTION
**Pull request by pengchengneo**
set up env
`uv sync --extra multimodal,tpu`

launch server

`uv run  python3 -u -m sgl_jax.launch_server --multimodal --model-path /models/Qwen/Qwen2.5-VL-7B-Instruct  --log-requests --disable-precompile --page-size 64`

run evalscope

`evalscope eval   --model Qwen2.5-VL-7B-Instruct   --api-url http://127.0.0.1:30000/v1/chat/completions   --api-key EMPTY   --eval-type openai_api   --datasets math_vision  --eval-batch-size 16`

test result
image

### CODE REVIEW

File: python/sgl_jax/srt/managers/tp_worker.py [hunk 1]
```diff
@@ -416,6 +416,7 @@ def generate_model_worker_batch(
             ),
             extend_input_logprob_token_ids=None,
             positions=np.concat([valid_positions, invalid_positions], axis=0),
+            mrope_positions=None,
```
Comment (pathfinder-pf) [inline python/sgl_jax/srt/managers/tp_worker.py:419]: maybe cache miss

Comment (pengchengneo) [reply python/sgl_jax/srt/managers/tp_worker.py:419]: it has been padding with _compute_mrope_positions_for_batch

Comment (pengchengneo) [reply python/sgl_jax/srt/managers/tp_worker.py:419]: qwen VL precompile logic is not implemented here, I think positions, input_embeddings should be padded together
